libres/lib/include/ert/enkf/enkf_util.hpp

```diff
@@ -35,6 +35,7 @@ extern "C" {
 
 void enkf_util_truncate(void *, int, ecl_data_type, void *, void *);
 double enkf_util_rand_normal(double, double, rng_type *rng);
+double enkf_util_rand_uniform(double a, double b, rng_type *rng);
```

Comment:
I think the `enkf_util_rand_uniform()` implementation was removed?

Reply:
Indeed, thanks.